lib/autoproj/daemon/github/client.rb

```diff
@@ -22,6 +24,20 @@ def pull_requests(owner, name, options = {})
                     end
                 end
 
+                # @return [Array<PushEvent, PullRequestEvent>]
+                def fetch_events(owner)
```

Comment:
I have the feeling that, all over the place, there are two distinct cases for both types of events. Maybe it would be better to separate the code paths from the source, instead of keeping them together and later on separating/dispatching them ?

Reply:
Not sure.. That would make sense now since we are only interested in PushEvent and PullRequestEvent but we will soon be fetching comments (for the hashtag thing) and maybe more later on.. We could group them here but I am not sure if that makes sense in the client API level..